sig-community-experience/meeting-template.md

```diff
@@ -0,0 +1,31 @@
+# Name of the Group/Governance Entity
+
+## Meta Information
+### Links, Resources, etc
+
+## Date & Time | Title in Calendar
+
+### Pre-meeting area
+#### Headline
+#### Main Topics to be discussed
+#### People/Teams/Skills needed
+#### Preparation
+#### Links to agenda, notes, slides, …
+
+### During meeting
+#### Attendees
+#### Agenda
+ - ...
+#### Notes
+ - ...
+#### Action items
+ -[ ] ...
+
+### Post-meeting area
+#### Headline
+#### Abstract
```

Comment:
I believe it would also be nice to provide guidance (in a comment) about how a good abstract looks like, e.g. "try to make this clear and concise, up to 3 sentences" or something along these lines